### DISCUSSION TITLE
Flashing BeagleBone

### DISCUSSION THREAD

**Discussion by Calorion** [kuestess/homebridge-platform-insteonlocal#261] [General]
Has anyone successfully flashed the BeagleBone in the Hub Pro with Debian? I've been using the instructions at https://beagleboard.org/getting-started and https://elinux.org/Beagleboard:BeagleBoneBlack_Debian#Flashing_eMMC, but it never successfully flashes. It does *something,* but no matter how long I leave it, it doesn't work on reboot. 

_Originally posted by @Calorion in https://github.com/kuestess/homebridge-platform-insteonlocal/issues/112#issuecomment-1146607164_

**Comment by 64Spaces** [comment]
It’s been a couple of years since I successfully flashed my Hub Pro. And it has since died. But I remember I had an issue with the board version number. I have two Hub Pro boxes with different board version numbers. The flashing would only work on the newer board.

**Comment by 64Spaces** [comment]
This is the old comment I had about my issue. In my case it was a Version 2 board versus Version 5 board.

https://github.com/kuestess/homebridge-platform-insteonlocal/issues/63#issuecomment-471225397

**Comment by wystewart** [comment]
debian 11 and 12 don't seem to work/boot on hub pro when booting from SD.  I can boot from SD using debian 10: https://forum.beagleboard.org/t/debian-10-x-buster-monthly-snapshot-2023-07-01-final/31203.

Haven't tried to flash eMMC yet, but I suspect can't use anything but 10 or older.

Per previous comment, check the sticker on the eMMC device on the beaglebone.  If is says V2, then you have to use the console image (1G), if it says V5, then you should be able to use the 4G IOT image.
